src/main/java/com/ericsson/ei/controller/FailedNotificationsControllerImpl.java

```diff
@@ -55,25 +55,25 @@
      */
     @Override
     @ApiOperation(value = "Retrieve failed notifications", response = QueryResponse.class)
-    public ResponseEntity<?> getFailedNotifications(@RequestParam("SubscriptionName") final String subscriptionName) {
+    public ResponseEntity<?> getFailedNotifications(@RequestParam("SubscriptionName") final String subscriptionNames) {
         ObjectMapper mapper = new ObjectMapper();
         QueryResponse queryResponse = new QueryResponse();
         QueryResponseEntity queryResponseEntity = new QueryResponseEntity();
         try {
-            List<String> response = processMissedNotification.processQueryMissedNotification(subscriptionName);
+            List<String> response = processMissedNotification.processQueryMissedNotification(subscriptionNames);
             if (!response.isEmpty()) {
                 queryResponseEntity = mapper.readValue(response.get(0), QueryResponseEntity.class);
             }
             queryResponse.setQueryResponseEntity(queryResponseEntity);
             LOGGER.debug("The response is : {}", response.toString());
-            if (processMissedNotification.deleteMissedNotification(subscriptionName)) {
-                LOGGER.debug("Failed notification with subscription name {} was successfully removed from database",
-                        subscriptionName);
+            if (processMissedNotification.deleteMissedNotification(subscriptionNames)) {
```

Comment:
There is a separate issue for fixing this in issue #354 . You can leave it as is for now, in case it affects tests...

Reply:
I couldnt find any logic behind deleting the missed notification. It might be deleted. However, as test need to be changed also, therefore, I am going to create a separate story/bug for it.